lib/app.jsx

```diff
@@ -473,7 +503,10 @@ export const App = connect( mapStateToProps, mapDispatchToProps )( React.createC
 					</ContextMenu>
 				}
 				{ state.authorized ?
-						<div className={mainClasses}>
+						<div className={mainClasses}
+							onMouseMove={ makeResizable( { isResizing, sourceListSelector: '.source-list', onResizable: this.props.setSourceListCanResize, onUpdateWidth: this.props.setSourceListWidth } ) }
+							onMouseDown={ startResizing( { canResize, onStartResizing: this.props.setSourceListResizing } ) }
+							onMouseUp={ stopResizing( { isResizing, onStopResizing: () => this.props.setSourceListResizing( false ) } ) } >
```

Comment:
`canResize` means their mouse is over the area where they can initiate dragging with a click.

`isResizing` means they are currently holding the mouse button down.

The `is-resizing` CSS class isn't being used so can be removed.